base/special/trig.jl

```diff
@@ -740,6 +740,37 @@ mulpi_ext(x::Float32) = DoubleFloat32(pi*Float64(x))
 mulpi_ext(x::Rational) = mulpi_ext(float(x))
 mulpi_ext(x::Real) = pi*x # Fallback
 
+@inline function sinpi_kernel(x::Float64) 
```

Comment:
Would be kind of nice to have a brief description of the method by which this works, similar to that of `acos()` on line 671.  Just a brief mention of something like "uses a taylor series expansion around 0" or something like that.